### PR TITLE
parameterized .exe filename ("" or "-bundle")

### PR DESCRIPTION
**Pull request by nickboldt**
parameterized .exe filename ("" or "-bundle"); readme addendum moved to #82

### CODE REVIEW

**Comment by nickboldt** [comment]
This is a combination of 3 of my previous performance & cleanup PRs plus one new one, which allows the build to produce TWO uniquely-named executables:

* DeveloperPlatformInstaller-win32-x64-stub.exe
* DeveloperPlatformInstaller-win32-x64-bundle.exe

File: README.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ In order to build the installer, you'll need to install some tools.
     npm install
     ```
 
+If either of the above steps fail, try deleting the c:\Users\<username>\.electron folder.
```
Comment (nickboldt) [inline README.md]: Note that I was getting strange errors trying to run builds and couldn't re-bootstrap the npm install. Found this:

https://github.com/electron-userland/electron-prebuilt/issues/96

And lo, I could build again.

**Comment by ppitonak** [comment]
@nickboldt I haven't noticed this PR before merging previous two. Please rebase and update this PR accordingly. For the future, please close old PR as soon as you create new one solving the same problem ;)

**Comment by nickboldt** [comment]
Rebased.

**Comment by dgolovin** [comment]
+1

**Comment by dgolovin** [comment]
The only thing I don like is 'stub' name, better just Setup-XXXXXX.exe because it is fully functional installer that downloads everything.

**Comment by nickboldt** [comment]
https://www.google.ca/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=define:+stub+installer 

But sure, we can call it ".exe" and "bundle.exe"

**Comment by nickboldt** [comment]
I've updated this PR to use '' or -bundle' rather than mentioning a stub.

That said I think we're missing a huge marketing opportunity here to co-brand with Brick Brewery (Waterloo, Ontario, Canada).

http://i.cbc.ca/1.2249059.1382678930!/httpImage/image.jpg_gen/derivatives/original_300/brick-cp-818394.jpg 

:D

**Comment by ppitonak** [comment]
Merged and Jenkins job updated.

**Pull request closed without merging**